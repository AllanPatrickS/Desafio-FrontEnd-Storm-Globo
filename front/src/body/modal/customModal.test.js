import { cleanup } from '@testing-library/react';
import { create } from "react-test-renderer";
import Modal from './customModal';

describe('Testing modal', () => {

  it('renders correctly', () => {
    const wrapper = create(<Modal open={false}/>)
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  afterEach(function () {
    cleanup();
  });

});