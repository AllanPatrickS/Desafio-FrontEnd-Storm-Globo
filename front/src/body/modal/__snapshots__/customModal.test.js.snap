// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing modal renders correctly 1`] = `null`;
